### PR TITLE
use non priviledged terrain from swisstopo

### PR DESCRIPTION
**Pull request by danduk82**
This fixes the VIP blocked terrain issue

### CODE REVIEW

File: src/viewer.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ export function setupViewer(container) {
   let terrainUrl;
   const ownTerrain = searchParams.get('ownterrain') !== 'false';
   if (ownTerrain) {
-    terrainUrl = 'https://terrain100.geo.admin.ch/1.0.0/ch.swisstopo.terrain.3d';
+    terrainUrl = 'https://3d.geo.admin.ch/1.0.0/ch.swisstopo.terrain.3d/default/20180601/4326/';
```
Comment (gberaudo) [inline src/viewer.js:96]: It does not look correct to use this old timestamp.

Comment (danduk82) [reply src/viewer.js:96]: it is the one officially advertised on api.geo.admin.ch

Comment (danduk82) [reply src/viewer.js:96]: we should summon and ask to @rebert

Comment (danduk82) [reply src/viewer.js:96]: I will send him a mail

Comment (gberaudo) [reply src/viewer.js:96]: Here is the URL/timestamp used by geoadmin:

GET | https://terrain101.geo.admin.ch/1.0.0/ch.swisstopo.terrain.3d/default/20200520/4326/8/269/194.terrain?v=1.0.0

It is also the one Tobias mentionned in the Cesium ticket back then.

Comment (danduk82) [reply src/viewer.js]: @gberaudo  in another NGM ticket I found a new timestamp: 20200520/4326/
I tried it and it seems to work

What do you think?

Comment (danduk82) [reply src/viewer.js]: seems the same to me

**Review by fredj — APPROVED** [review]

**Pull request merged**